src/custom/CortiAuth.ts

```diff
@@ -158,28 +202,31 @@ export class Auth extends FernAuth {
              * Patch: removed `requestType: "json"`, made body a URLSearchParams object
              */
             body: new URLSearchParams({
-                ...serializers.AuthGetTokenRequest.jsonOrThrow(request, {
-                    unrecognizedObjectKeys: "strip",
-                    omitUndefined: true,
-                }),
+                // Filter out empty client_secret for PKCE flow
+                ...Object.fromEntries(
+                    Object.entries(validatedRequest).filter(([key, value]) => 
+                        !(isPkce && key === "client_secret" && !value)
+                    )
+                ),
                 scope: "openid",
                 /**
                  * Patch: `grant_type` uses values from request or defaults to "client_credentials"
                  */
-                grant_type: request.grantType || "client_credentials",
+                grant_type: grantType,
                 /**
                  * Patch: added `code` and `redirect_uri` fields for Authorization code flow
                  * Patch: added `refresh_token` field for Refresh token flow
                  */
-                ...(request.grantType === "authorization_code"
+                ...(grantType === "authorization_code"
                     ? {
-                        code: request.code,
-                        redirect_uri: request.redirectUri
+                        code: (request as any).code,
+                        redirect_uri: (request as any).redirectUri,
+                        ...((request as any).codeVerifier ? { code_verifier: (request as any).codeVerifier } : {})
```

Comment:
let's just pass it every time. `undefined` shouldn't break anything in theory (although for `URLSearchParams` I'm not sure, needs to be checked)